cmd/argocd/commands/app.go

```diff
@@ -1375,7 +1334,7 @@ func waitOnApplicationStatus(acdClient apiclient.Client, appName string, timeout
 	}
 
 	w := tabwriter.NewWriter(os.Stdout, 5, 0, 2, ' ', 0)
-	fmt.Fprintf(w, waitFormatString, "TIMESTAMP", "GROUP", "KIND", "NAMESPACE", "NAME", "STATUS", "HEALTH", "HOOK", "MESSAGE")
+	_, _ = fmt.Fprintf(w, waitFormatString, "TIMESTAMP", "GROUP", "KIND", "NAMESPACE", "NAME", "STATUS", "HEALTH", "OPERATION STATE", "MESSAGE")
```

Comment:
Don't use spaces in CLI table headers. It confuses awk. Standard practice is to hyphenate if necessary.

Reply:
Reverted.